kork/kork-core/src/main/java/com/netflix/spinnaker/kork/yaml/YamlHelper.java

```diff
@@ -0,0 +1,96 @@
+package com.netflix.spinnaker.kork.yaml;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.yaml.snakeyaml.DumperOptions;
+import org.yaml.snakeyaml.LoaderOptions;
+import org.yaml.snakeyaml.Yaml;
+import org.yaml.snakeyaml.constructor.Constructor;
+import org.yaml.snakeyaml.constructor.SafeConstructor;
+import org.yaml.snakeyaml.representer.Representer;
+import org.yaml.snakeyaml.resolver.Resolver;
+
+@Component
+public class YamlHelper {
```

Comment:
javadoc please

Reply:
@dbyron-sf here is the change https://github.com/rahul-chekuri/spinnaker/commit/6495d094